rcode-reader/src/main/java/eu/rssw/pct/elements/VariableElement.java

```diff
@@ -0,0 +1,77 @@
+package eu.rssw.pct.elements;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.util.Set;
+
+import eu.rssw.pct.AccessType;
+import eu.rssw.pct.DataType;
+import eu.rssw.pct.RCodeInfo;
+
+public class VariableElement extends AbstractAccessibleElement {
+  private static final int READ_ONLY = 1;
+  private static final int WRITE_ONLY = 2;
+  private static final int BASE_IS_DOTNET = 4;
+  private static final int NO_UNDO = 8;
+
+  private int data_type;
+  // private int recordPosition;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3ACommentedOutCodeLine)